qt/aqt/addons.py

```diff
@@ -820,3 +850,65 @@ def accept(self):
 
         self.onClose()
         super().accept()
+
+
+# .ankiaddon installation wizard
+######################################################################
+
+
+def installAddonPackages(
+    addonsManager: AddonManager,
+    paths: List[str],
+    parent: QWidget = None,
+    external: bool = False,
+) -> bool:
+
+    if external:
+        names_str = ",<br>".join(f"<b>{os.path.basename(p)}</b>" for p in paths)
+        q = _(
+            "<b>Important</b>: As add-ons are programs downloaded from the internet, "
+            "they are potentially malicious."
+            "<b>You should only install add-ons you trust.</b><br><br>"
+            "Are you sure you want to proceed with the installation of the "
+            f"following add-on(s)?<br><br>{names_str}<i>"
```

Comment:
You can't use an f-string inside _(), as it will prevent the string from being translated. The string will need to %s instead, with the replace done to the output of _()

Reply:
Thanks for the pointer! Updated.